### ISSUE TITLE
line-height is defined at html and body with differents values

### ISSUE THREAD

**Issue by rentalhost** [twbs/bootstrap#30645]
Same as old https://github.com/twbs/bootstrap/issues/24964

The `bootstrap-reboot.css` defines:

```css
html {
  /** ... */
  line-height: 1.15;
  /** ... */
}

/** ... */

body {
  /** ... */
  line-height: 1.5;
  /** ... */
}
```

And it seems non-sense. The original post, @mdo said that *It's part of Reboot and there as part of a global browser reset, forked from Normalize. Leaving for now*.

It is not part of `normalize.css` currently, but I do not know if before was, anyway, maybe is time to fix it?

**Comment by mdo** [comment]
This was fixed in #28698 and is coming in v5.

**Comment by GoodSuperiorbedding** [comment]
https://github.com/twbs/bootstrap/issues/30645#issue-606540381
